card.py

```diff
@@ -0,0 +1,16 @@
+import random
+from color import Color
+
+
+class Card:
+    def __init__(self):
+        self.num = random.randrange(1, 11)
+        colors = list(Color.__members__.values())
+        self.color = random.choice(colors).name
+
+    def __eq__(self, other):
```

Comment:
This function is called when executing this line:   **if card == pulled_card**

Both **card** and **pulled_card** are Card class objects.
The line **card == pulled_card** is actually identical to:  **card.__eq__(self, pulled_card)**

We need to implement __eq__ because we're dealing with a new class we created, so we need to define how the comparison between two class objects work.